example/src/main/res/values/colors.xml

```diff
@@ -15,4 +15,6 @@
     <color name="orange_text">#E75B30</color>
     <color name="orange_normal">#DF552B</color>
     <color name="orange_dark">#AA4020</color>
+
+    <color name="survey_text">#FFFFFF</color>
```

Comment:
```suggestion
```

Reply:
I'd like to keep the example app as it is for now. The other adjustments are fine!